webpack/assets/javascripts/react_app/routes/common/LoadingPage/index.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { Spinner } from 'patternfly-react';
+import './loadingpage.scss';
+
+const LoadingPage = () => (
+  <div id="loading-page">
+    <Spinner size="lg" loading />
```

Comment:
why not use loadingState here?

Reply:
loadingState needs children and handles which component to render if i'm not mistaken, the idea of this file is just to be a "stupid" default skeleton

Reply:
I still like the fact that you can show the spinner after a certain amount of ms and not to show it if its too fast, I wonder if can't just use it without childrens?

Reply:
I added `loadingState` with 500ms delay for the default Loading page, its great for spinners.
For custom skeletons, I think its fine to show them right away.